### PR TITLE
Fixes a bunch of bugs related to linkify and stuff

### PR DESCRIPTION
**Pull request by Discordius**
This vendors two files from the draft-js-anchor-plugin that we already depended on

### CODE REVIEW

File: packages/lesswrong/components/async/EditorForm.jsx [hunk 1]
```diff
@@ -79,7 +80,7 @@ class EditorForm extends Component {
       { button: BoldButton,                    commentEditor: true   },
       { button: ItalicButton,                  commentEditor: true   },
       { button: UnderlineButton,               commentEditor: true   },
-      { button: linkPlugin.LinkButton,         commentEditor: true   },
+      { button: LinkButton,             commentEditor: true   },
```
Comment (jimrandomh) [inline packages/lesswrong/components/async/EditorForm.jsx]: Misaligned whitespace

**Comment by jimrandomh** [comment]
Flagging that this is a high-risk PR, given (a) that it's draft-JS, and (b) that it's trying to fix a bug inside a vendored library by commenting out something poorly understood.

File: packages/lesswrong/components/async/editor-plugins/linkifyPlugin.js [hunk 2]
```diff
@@ -167,23 +168,26 @@ const createLinkifyPlugin = () => {
             }),
           });
         } else {
+          // LESSWRONG: This branch was causing a bunch of problems and is pretty buggy, so I deactivated it
+          
           // We are no longer in a URL but the entity is still present. Remove it from
           // the current character so the linkifying "ends".
-          const entityRange = new SelectionState({
```
Comment (jimrandomh) [inline packages/lesswrong/components/async/editor-plugins/linkifyPlugin.js]: I'd say the main thing that needs testing here is all the conditions in which you're typing a linkified link, and then aren't anymore. Eg: you type `http://www.example.com<space>`, or you put your cursor in the middle and type `http://www.exa<space>mple.com`.

**Comment by Discordius** [comment]
Yeah, I tested a lot of those cases. My guess is that it's best for us to merge this into devel and use it for at least a few days before pushing it live.

**Pull request merged**